charts/rstudio/templates/deployment.yml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: r-studio
```

Comment:
Resource names should be populated from the 'fullname' template helper, e.g. `name: {{ template "fullname" . }}`